edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/EdgeMessage.cs

```diff
@@ -34,18 +42,24 @@ public bool Equals(EdgeMessage other)
             }
 
             return this.Body.SequenceEqual(other.Body) &&
-                this.Properties.Keys.Count == other.Properties.Keys.Count &&
-                this.Properties.Keys.All(key => other.Properties.ContainsKey(key) && Equals(this.Properties[key], other.Properties[key])) &&
-                this.SystemProperties.Keys.Count == other.SystemProperties.Keys.Count &&
-                this.SystemProperties.Keys.All(skey => other.SystemProperties.ContainsKey(skey) && Equals(this.SystemProperties[skey], other.SystemProperties[skey]));
+                   this.Properties.Keys.Count == other.Properties.Keys.Count &&
+                   this.Properties.Keys.All(key => other.Properties.ContainsKey(key) && Equals(this.Properties[key], other.Properties[key])) &&
+                   this.SystemProperties.Keys.Count == other.SystemProperties.Keys.Count &&
+                   this.SystemProperties.Keys.All(skey => other.SystemProperties.ContainsKey(skey) && Equals(this.SystemProperties[skey], other.SystemProperties[skey]));
         }
 
         public override bool Equals(object obj)
         {
             if (ReferenceEquals(null, obj))
+            {
```

Comment:
Is requiring curly braces also a rule? Doesn't seem to apply to Build method below.

Reply:
yes, there is a format rule.  I prefer to use curly braces all the time.  do you prefer to allow simple-if statement without curly brace?

Reply:
I don't mind it... I also usually always use curly braces.

Reply:
This provides by Resharper formatting; and this is not enforced in StyleCop.